### PR TITLE
Or 1810 update the supporting of the native token feature for optimism portal2.sol

### PR DESCRIPTION
**Pull request by nguyenzung**

### CODE REVIEW

File: op-bindings/bindings/uniswapv3factory_more.go [hunk 1]
```diff
@@ -23,5 +23,5 @@ func init() {
 
 	layouts["UniswapV3Factory"] = UniswapV3FactoryStorageLayout
 	deployedBytecodes["UniswapV3Factory"] = UniswapV3FactoryDeployedBin
-	immutableReferences["UniswapV3Factory"] = false
+	immutableReferences["UniswapV3Factory"] = true
```
Comment (rlgns98kr) [inline op-bindings/bindings/uniswapv3factory_more.go:26]: Can you restore this manually? (@AaronLee22 set this value manually previos!)

**Comment by boohyung** [comment]
@nguyenzung Do you have a plan for solving conflict after internal code review?

**Comment by nguyenzung** [comment]
> @nguyenzung Do you have a plan for solving the conflicts after internal code review?

Yep. I think there are similar conflicts on the branch codeReview, so I will resolve and the the updates for internal audit first, then will merge OptimismPortal2.sol

**Comment by nguyenzung** [comment]
I will make other PR to merge this branch into OR-1843-merge-internal-audit-to-main.
Then other PR for merging OR-1843-merge-internal-audit-to-main to the main branch

**Pull request closed without merging**